src/clu_log.c

```diff
@@ -77,3 +74,39 @@ void wolfCLU_Log(int logLevel, const char *const fmt, ...)
     if (logFunction)
         logFunction(logLevel, msgStr);
 }
+
+
+void wolfCLU_LogErrorQueue(void)
+{
+    int err;
+    const char* file;
+    int line = 0;
+
+    while ((err = wolfSSL_ERR_get_error_line_data(&file, &line,
+                                                  NULL, NULL)) != 0) {
+        if (file != NULL) {
+            WOLFCLU_LOG(WOLFCLU_E0, "Error %s:%d: %s (%d)", file, line,
```

Comment:
in other places ` wolfCLU_LogError` is used over WOLFCLU_LOG(WOLFCLU_E0, this should match the other calls to be consistent